code/game/objects/structures/flora.dm

```diff
@@ -91,6 +91,10 @@
 	icon_state = "[icon_state][rand(1, 6)]"
 	..()
 
+/obj/structure/flora/tree/jungle/small
+	pixel_y = 0
+	pixel_x = -32

```

Comment:
no, he's commenting that there' no ```icon_state = ...``` line, meaning the small tree will look like a big tree